cypress/e2e/clue/branch/teacher_tests/teacher_network_spec.js

```diff
@@ -36,9 +36,9 @@ describe('Networked dividers for networked teacher', () => {
     teacherNetwork.verifyDividerLabel('workspaces', 'my-classes');
     teacherNetwork.verifyDividerLabel('workspaces', 'my-network');
 
-    cy.openSection('my-work', 'starred');
-    teacherNetwork.verifyDividerLabel('starred', 'my-classes');
-    teacherNetwork.verifyDividerLabel('starred', 'my-network');
+    // cy.openSection('my-work', 'starred');
+    // teacherNetwork.verifyDividerLabel('starred', 'my-classes');
+    // teacherNetwork.verifyDividerLabel('starred', 'my-network');
```

Comment:
We should include a TODO or some other notes in tests when we have to disable them. The hope is that some day we'll be able to get the tests working again, but who knows when that day will come.